02-express-tutorial/answers/final/004-express-app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.static('./Public'));
+
+app.get('/',(req,res)=>{
+    res.sendFile(path.resolve(__dirname,'./navbar-app/index.html'))
```

Comment:
Using `path.resolve` is definitely the right way to refer to file paths, but in this case the resolved path doesn't exist (it resolves to `02-express-tutorial/answers/final/navbar-app/index.html`). You need to go up two parent directories to refer to the correct path => `../../navbar-app/index.html`.

Reply:
Hi Taylor. Thank you so much for reviewing my code. Indeed, you are completely right, and upon implementing this correction, I am now rendering the page on server 5000 correctly. I must have been working on the code in the 'app.js' file that exists two parent directories above the current /answers/final/004-express-app.js file. I have now corrected the code, and upon running 'node 004-express-app.js" (while ensuring that my terminal shows I am connected to the path ../**answers/final**') I see that the website renders correctly.